.env.example

```diff
@@ -0,0 +1,3 @@
+NODE_ENV=production
```

Comment:
Might be a personal view but what about creating a .env file with local options, and add remote examples in comments. So we don't bother with the example file.

So we don't get annoyed with a locally (on dev side) changed file, would the following work: Commit the .env file and then add it to .gitignore ?

Reply:
I think its easy enough to create a .env file & the process is well enough documented in the README.md. Also, I think the .env file is already added to the .gitignore.

Reply:
That's fair, yeah. I'll do that. Also I honestly don't think we need `NODE_ENV` in it at all since webpack sets that for us iirc

Reply:
oh, @gildub the problem with committing .env is that when you modify your local one to add the real API URL it hangs around as an uncommitted change, and we may accidentally commit that. I think I'm going to keep the .env.example for that reason. When we move that to a run-time thing maybe we can change it.